### PR TITLE
Remove the PHPdoc from the impl class

### PR DESCRIPTION
**Pull request by umpirsky**
As suggested in https://github.com/sonata-project/GoogleAuthenticator/pull/111#discussion_r203344998.

Also removed deprecated method from the interface.

### CODE REVIEW

File: src/GoogleAuthenticator.php [hunk 1]
```diff
@@ -57,8 +57,7 @@ public function __construct(int $passCodeLength = 6, int $secretLength = 10, \Da
     }
 
     /**
-     * @param string $secret
-     * @param string $code
+     * {@inheritdoc}
```
[the remark this answers is not in the document]
Comment (umpirsky) [reply src/GoogleAuthenticator.php]: Fixed.

File: src/GoogleAuthenticator.php [hunk 2]
```diff
@@ -123,15 +113,6 @@ public function getCode($secret, /* \DateTimeInterface */$time = null): string
...
 
-    /**
-     * NEXT_MAJOR: Remove this method.
-     *
-     * @param string $user
-     * @param string $hostname
-     * @param string $secret
-     *
-     * @deprecated deprecated as of 2.1 and will be removed in 3.0. Use Sonata\GoogleAuthenticator\GoogleQrUrl::generate() instead.
-     */
```
Comment (core23) [inline src/GoogleAuthenticator.php]: You should keep this when removing the method from the interface

Comment (umpirsky) [reply src/GoogleAuthenticator.php]: Fixed, thanks!

Comment (umpirsky) [reply src/GoogleAuthenticator.php]: So, what should we do?

**Comment by umpirsky** [comment]
Resolved conflicts.

File: src/GoogleAuthenticator.php [hunk 3]
```diff
@@ -86,12 +82,6 @@ public function checkCode($secret, $code): bool
         return $result > 0;
     }
 
-    /**
-     * NEXT_MAJOR: add the interface typehint to $time and remove deprecation.
```
Comment (greg0ire) [inline src/GoogleAuthenticator.php]: Your commit message says you removed phpdoc, yet you also remove this valuable piece of info, why?

File: src/GoogleAuthenticatorInterface.php [hunk 4]
```diff
@@ -29,16 +29,5 @@ public function checkCode($secret, $code): bool;
...
-    /**
-     * NEXT_MAJOR: Remove this method.
-     *
-     * @param string $user
-     * @param string $hostname
-     * @param string $secret
-     *
-     * @deprecated deprecated as of 2.1 and will be removed in 3.0. Use Sonata\GoogleAuthenticator\GoogleQrUrl::generate() instead.
-     */
-    public function getUrl($user, $hostname, $secret): string;
```
Comment (greg0ire) [inline src/GoogleAuthenticatorInterface.php]: That is a BC-break, isn't it?

**Comment by umpirsky** [comment]
@greg0ire This was commited 6 months ago, I don't remember. Maybe it's best to close pull request and let someone do it again if it's still relevant.

**Pull request closed without merging**